### PR TITLE
Release v1.13.0-snapshot.20210419.6730.0.8c3a8c04

### PR DESCRIPTION
**Pull request by kamil-da**
### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [x] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: LATEST [hunk 1]
```diff
@@ -1,3 +1,4 @@
+8c3a8c042840ffd0260ea992fab6fedc3915572d 1.13.0-snapshot.20210419.6730.0.8c3a8c04
 2dc09ba2def7efcdeba10c8849dec0c671b85b3a 1.13.0-snapshot.20210413.6706.0.2dc09ba2
```
Comment (cocreature) [inline LATEST:2]: Please overwrite the existing 1.13.0 snapshot. The idea is to keep one snapshot per stable version until we finally replace it by the stable version and add a new line for the snapshot for the next stable version.

Comment (kamil-da) [reply LATEST:2]: Ah, got it!

Comment (kamil-da) [reply LATEST]: Fixed 👍

**Comment by kamil-da** [comment]
Manual tests passed on macOS.

**Comment by mziolekda** [comment]
Manual tests passed on Windows.

**Pull request merged**